src/constants.F90

```diff
@@ -139,6 +139,9 @@ module constants
   ! Maximum number of lost particles
   integer, parameter :: MAX_LOST_PARTICLES = 10
 
+  ! Maximum number of lost particles, relative to the total number of particles
+  real(8), parameter :: REL_MAX_LOST_PARTICLES = 1e-5_8
```

Comment:
I personally think 1e-5 is too high actually. I'd feel better with one in a million.